Error message when insufficient permissions

[2020-05-11T17:35:08.721] [ERROR] Error joining voice connection. cached = false. song = Lucky Strike:Dreamcatcher:AM7ro_nBCHI err = Error [VOICE_JOIN_CHANNEL]: You do not have permission to join this voice channel.

--- a/helpers/utils.js
+++ b/helpers/utils.js
@@ -257,6 +257,9 @@ const playSong = async (gameSession, guildPreference, db, message, client) => {
         }
         catch (err) {
             logger.error(`${getDebugContext(message)} | Error joining voice connection. cached = ${gameSession.isSongCached}. song = ${gameSession.getDebugSongDetails()} err = ${err}`);
+            sendErrorMessage(message, "Missing voice permissions", "The bot is unable to join the voice channel you are in.");
+            gameSession.endRound();
+            return;
         }
     }
     // We are unable to pipe the above ytdl stream into Discord.js's play

